sovtoken/sovtoken/test/auth_map/test_auth_mint.py

```diff
@@ -16,19 +18,19 @@ def mint_tokens(helpers, addresses):
 
 def steward_do_mint(helpers, outputs):
     """ Sends and check a mint txn """
-    outputs_ready = helpers.request._prepare_outputs(outputs)
+    request = helpers.request.mint(outputs)
 
-    payload = {
-        TXN_TYPE: MINT_PUBLIC,
-        OUTPUTS: outputs_ready,
-    }
-    identifier = helpers.wallet._steward_wallets[0].defaultId
+    request.signatures = {}
```

Comment:
Do we need to do this because `mint` signs by 3 Trustees by default?

Reply:
Yes. We can introduce more flexible logic in future